salt/pillar/profiles.sls

```diff
@@ -1,5 +1,11 @@
 profiles:
-    base_url: http;//dev--profiles.elifesciences.org
+    #oauth_clients:
```

Comment:
could still leave one fake client configured in Vagrant. Helps finding out errors inside the template loop